cogs/general.py

```diff
@@ -655,7 +655,7 @@ async def blacklist_add(self, ctx, *, args):
         args = args.replace('\n', ' ').split()
         list_of_ids = []
         reason = "None"
-        for arg_index in range(len(args)):
+        for arg_index, args in enumerate(args):
```

Comment:
This should be `for arg_index, arg in enumerage(args)`. As it is now, `args` is doubly-defined.

Reply:
Fixed here 

https://github.com/r-hensley/Rai/commit/f82944fc6cc0bbd77775cea0e1051bc3292a67ff#diff-7d60aa466f8080cbbc2ad475ee3d72248d327ce3dd62e13ea4fb435706dcbeb6L645-R659